lxd/main_init.go

```diff
@@ -40,17 +39,20 @@ func cmdInit() error {
 	imagesAutoUpdate = true
 
 	backendsAvailable := []string{"dir"}
-	backendsSupported := []string{"dir", "zfs"}
 
-	// Detect zfs
-	out, err := exec.LookPath("zfs")
-	if err == nil && len(out) != 0 && !runningInUserns {
-		_ = loadModule("zfs")
+	// Check available backends
+	for _, driver := range supportedStoragePoolDrivers {
```

Comment:
Note to self: Needs to account for running in userns where only dir needs to be made available.